lib/matplotlib/__init__.py

```diff
@@ -764,6 +763,14 @@ def __getitem__(self, key):
                 from matplotlib import pyplot as plt
                 plt.switch_backend(rcsetup._auto_backend_sentinel)
 
+        elif key == "path.effects" and self is globals().get("rcParams"):
+            # defers loading of patheffects to avoid circular imports
+            import matplotlib.patheffects as path_effects
+            # use patheffects object or instantiate patheffects.object(**kwargs)
+            return [pe if isinstance(pe, path_effects.AbstractPathEffect)
+                       else getattr(path_effects, pe[0])(**pe[1])
+                    for pe in self._get('path.effects')]
```

Comment:
This needs to support third party patheffects too (e.g. `pkgname.modulename.patheffect` instead of just `withStroke`)